crypto/batchverifier.go

```diff
@@ -112,8 +116,10 @@ func (b *BatchVerifier) GetNumberOfEnqueuedSignatures() int {
 // Verify verifies that all the signatures are valid. in that case nil is returned
 // if the batch is zero an appropriate error is return.
 func (b *BatchVerifier) Verify() error {
-	if b.GetNumberOfEnqueuedSignatures() == 0 {
+	if cnt := b.GetNumberOfEnqueuedSignatures(); cnt == 0 {
 		return ErrZeroTransactionInBatch
+	} else if cnt < minBatchVerifierBatchSize {
```

Comment:
This condition is already been implemented in the C code. I'm curious what is the effect of this change. can you add a benchmark?

Reply:
Good idea ... let me see if we have one, I'll write one